seqr/views/apis/report_api.py

```diff
@@ -173,6 +178,9 @@ def _add_row(row, family_id, row_type):
         }},
     )
 
+    for row in parsed_rows[SUBJECT_ROW_TYPE]:
+        row.update(family_diseases[row['family_id']])
```

Comment:
`family_diseases` gets populated if `row_type == FAMILY_ROW_TYPE`, so is there ever possibility that `family_diseases` is empty and cause an attribute error here?

Reply:
no, in `parse_anvil_metadata` we always call ` add_row` with `FAMILY_ROW_TYPE` before we ever call `add_row` with `SUBJECT_ROW_TYPE`. There are a couple of other places where we rely on this strict prdering